### ISSUE TITLE
[Bug]: Slow startup and freeze installing steam

### ISSUE THREAD

**Issue by chinh4thepro** [bottlesdevs/Bottles#2048]
### Describe the bug

When I startup bottles it takes ~30-45 minutes to start up. When I try to install steam via the installers menu it reports bottles is not responding, and it takes a while to install windows dependencies. I have tried this on fedora and arch with the same behaviors.

### To Reproduce

First Problem:
1. Open bottles

Second Problem:
1. Go into a bottle
2. Click on installers
3. Click on steam
4. Spam wait on the not responding prompt

### Package

Flatpak from Flathub

### Distribution

Fedora 36

### Troubleshooting Logs

```shell
Version: 2022.8.28-brescia-2
Display:
    X.org: true
    X.org (port): :99.0
    Wayland: true
Graphics:
    vendors:
        amd:
            vendor: amd
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/radeon_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/radeon_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 5.19.7-203.fsync.fc36.x86_64
Distro:
    Name: GNOME
    Version: '"42 (Flatpak runtime)"'
Disk:
    Total: 9357025280
    Free: 9356877824
RAM:
    MemTotal: 17.4GiB
    MemAvailable: 10.6GiB
Bottles_envs: null
```

### Display Protocol

Wayland

### Display Backend used by Bottles

Wayland

**Comment by mirkobrombin** [comment]
Start bottles from terminal, then try again and post the output here:
flatpak run com.usebottles.bottles

**Comment by lukebemish** [comment]
I can reproduce the slow startup times on Pop!_OS 22.04. X11 instead of Wayland in my case. It sometimes takes something like 20-30 minutes to initially boot (and to do everything after that, in fact), which makes the program basically unusable. Checking something like htop in that time, and nothing on my system is actually _doing_ anything.

Troubleshooting logs:
```
Version: 2022.8.28-brescia-2
Display:
    X.org: true
    X.org (port): :99.0
    Wayland: false
Graphics:
    vendors:
        intel:
            vendor: intel
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/intel_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/intel_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 5.19.0-76051900-generic
Distro:
    Name: GNOME
    Version: '"42 (Flatpak runtime)"'
Disk:
    Total: 8308781056
    Free: 8308629504
RAM:
    MemTotal: 15.5GiB
    MemAvailable: 10.5GiB
Bottles_envs: null
```

Terminal log during startup:
```
15:50:37 (INFO) Bottles Started! 
16:05:59 (INFO) Catalog components loaded 
16:08:10 (INFO) Catalog installers loaded 
16:10:21 (INFO) Catalog dependencies loaded 
16:10:21 (INFO) Performing Bottles checks… 
16:10:21 (INFO) Dxvks found:
	 - dxvk-1.10.3
 
16:10:21 (INFO) Vkd3ds found:
	 - vkd3d-proton-2.6-1-5b73139
 
16:10:21 (INFO) Nvapis found:
	 - dxvk-nvapi-v0.5.4
 
16:10:21 (INFO) Latencyflexs found:
	 - latencyflex-v0.1.0
 
16:10:21 (INFO) Runners found:
	 - sys-wine-7.0
	 - soda-7.0-3
 
16:10:38 (INFO) Bottles found:
	 - ChemDraw
 
16:10:38 (INFO) Library saved 
```

**Comment by mirkobrombin** [comment]
Is the steam proton manager feature enabled?

**Comment by mirkobrombin** [comment]
Maybe duplicate of 
https://github.com/bottlesdevs/Bottles/issues/2021

**Comment by chinh4thepro** [comment]
18:58:28 (INFO) Bottles Started! 
19:13:50 (INFO) Catalog components loaded 
19:16:01 (INFO) Catalog installers loaded 
19:18:12 (INFO) Catalog dependencies loaded 
19:18:12 (INFO) Performing Bottles checks… 
19:18:12 (INFO) Dxvks found:
	 - dxvk-1.10.3
 
19:18:12 (INFO) Vkd3ds found:
	 - vkd3d-proton-2.6-1-5b73139
 
19:18:12 (INFO) Nvapis found:
	 - dxvk-nvapi-v0.5.4
 
19:18:12 (INFO) Latencyflexs found:
	 - latencyflex-v0.1.0
 
19:18:12 (INFO) Runners found:
	 - sys-wine-7.0
	 - soda-7.0-3
	 - wine-ge-7-27
 
19:18:30 (INFO) Bottles found:
	 - gaming
 
19:18:30 (INFO) Library saved

**Comment by chinh4thepro** [comment]
seems like it takes the same amount of time even with the FORCE_OFFLINE=1 argument

**Comment by chinh4thepro** [comment]
seems like backend error

**Comment by chinh4thepro** [comment]
strange I had changed my name server to 1.1.1.1 in /etc/resolv.conf and it seems to have fixed this problem
